packages/flutter/lib/src/rendering/box.dart

```diff
@@ -806,12 +805,12 @@ class BoxHitTestResult extends HitTestResult {
   ///  * [addWithPaintTransform], which accomplishes the same thing, but takes a
   ///    _paint_ transform matrix.
   bool addWithRawTransform({
-    @required Matrix4 transform,
-    @required Offset position,
-    @required BoxHitTest hitTest,
+    required Matrix4? transform,
+    required Offset? position,
```

Comment:
non-nullable position here too

Reply:
But the doc explicitally mentions null:

```
  /// The `position` argument may be null, which will be forwarded to the
  /// `hitTest` callback as-is. Using null as the position can be useful if
  /// the child speaks a different hit test protocol then the parent and the
  /// position is not required to do the actual hit testing in that protocol.
```

Reply:
Yeah, I'm proposing removing that and replacing it with dedicated methods to handle that case.